fpm/recipes/govuk-python-3.6.12/recipe.rb

```diff
@@ -1,6 +1,7 @@
-class Python36 < FPM::Cookery::Recipe
+class GovukPython3612 < FPM::Cookery::Recipe
   homepage 'https://www.python.org/downloads/release/python-3612/'
-  name 'python-3.6.12'
+  name 'govuk-python'
+  version '3.6.12'
   
   source 'https://www.python.org/ftp/python/3.6.12/Python-3.6.12.tgz'
```

Comment:
Might as well substitute the version in now that it's a variable.

```suggestion
  source 'https://www.python.org/ftp/python/#{version}/Python-#{version}.tgz'
```